cs-utilities/src/main/java/io/cloudslang/content/utilities/actions/DefaultIfEmptyAction.java

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************************
+ * (c) Copyright 2017 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.content.utilities.actions;
+
+import com.hp.oo.sdk.content.annotations.Action;
+import com.hp.oo.sdk.content.annotations.Output;
+import com.hp.oo.sdk.content.annotations.Param;
+import com.hp.oo.sdk.content.annotations.Response;
+import com.hp.oo.sdk.content.plugin.ActionMetadata.MatchType;
+import com.hp.oo.sdk.content.plugin.ActionMetadata.ResponseType;
+import io.cloudslang.content.constants.ReturnCodes;
+import io.cloudslang.content.utils.BooleanUtilities;
+import io.cloudslang.content.utils.StringUtilities;
+
+import java.util.Map;
+
+import static io.cloudslang.content.constants.OutputNames.*;
+import static io.cloudslang.content.constants.ReturnCodes.*;
+import static io.cloudslang.content.utils.OutputUtilities.getFailureResultsMap;
+import static io.cloudslang.content.utils.OutputUtilities.getSuccessResultsMap;
+
+/**
+ * Created by moldovai on 8/21/2017.
+ */
+public class DefaultIfEmptyAction {
+
+    /**
+     * This operation checks if a string is blank or empty and if it's true a default value
+     * will be assigned instead of the initial string.
+     *
+     * @param initial_value The initial string.
+     * @param default_value The default value used to replace the initial string.
+     * @param trim A variable used to check if the initial string is blank or empty.
+     * @return a map containing the output of the operation. Keys present in the map are:
+     * <p/>
```

Comment:
why do you use "\<p/>", "\<br>", "\<b>" or any ... ?

Reply:
I've seen it in another "Action" file and used that form